sql/core/src/test/scala/org/apache/spark/sql/test/SharedSQLContext.scala

```diff
@@ -17,7 +17,7 @@
 
 package org.apache.spark.sql.test
 
-import org.apache.spark.sql.{ColumnName, SQLContext}
+import org.apache.spark.sql.{Column, ColumnName, SQLContext}
```

Comment:
This is an irrelevant change, added to stop IntelliJ IDEA highlighting errors in ScalaDoc.